tools.js

```diff
@@ -43,6 +43,91 @@ class Tools {
     return url;
   }
 
+  /**
+   * Takes an Invitation or Group ID and returns a pretty version of it.
+   * 
+   * @param {string} id - Invitation or Group ID.
+   * @param {boolean} onlyLast - If true, only the last part of the ID will be returned.
+   * @returns {string} Pretty version of the ID.
+   */
+  prettyId(id, onlyLast) {
+    if (!id) {
+      return '';
+    } else if (id.indexOf('~') === 0 && id.length > 1) {
+      return id.substring(1).replace(/_|\d+/g, ' ').trim();
+    } else if (id === 'everyone' || id === '(anonymous)' || id === '(guest)' || id === '~') {
+      return id;
+    }
+
+    const lowercaseExceptions = [
```

Comment:
@zbialecki should we keep this lowercase exceptions?

Reply:
Yeah I don't think we actually need it anymore. I think all IDs are using uppercase versions of those strings.

Reply:
OK, I'll remove the check